icecreamshop/src/main/java/com/summerschool/icecreamshop/model/Category.java

```diff
@@ -0,0 +1,52 @@
+package com.summerschool.icecreamshop.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private String name;
+
+    @Column(length = 500)
+    private String description;
+
```

Comment:
Lipseste lista de produse de aici - asa cum in clasa Category in librarie aveam:    
@OneToMany(mappedBy = "category", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
 private Set<Book> books = new HashSet<>();